src/app/shared/services/signalR/signal-r.service.ts

```diff
@@ -29,7 +29,6 @@ export class SignalRService {
 
     this.hubConnection
       .start()
-      .then(() => console.log('Connection started'))
```

Comment:
let's save this one, so we can track successful connection, and the rest is good!